app/config/sources.inc.php

```diff
@@ -409,6 +409,14 @@
         ],
         'supported_locales' => $firefox_locales,
     ],
+    'firefox/whatsnew_76.lang' => [
+        'deadline'          => '2020-04-30',
```

Comment:
The deadline should be 4/24 this Friday. The completed locale list is due to releng 10 days before the launch. Please change.

Reply:
Could you confirm what the fallback page is if WNP76 is incomplete for non-P1 locales?

Reply:
It would be our standard fallback page: https://www.mozilla.org/en-US/firefox/whatsnew/all/